<launch>

	<!-- PID gains -->
	<arg name="p_gain" default="50"/>
	<arg name="i_gain" default="0"/>
	<arg name="d_gain" default="20"/>

	<!-- Participant ID (1 -- max) -->
	<arg name="ID" default="0"/>

	<!-- Task # (0-3) -->
	<arg name="task" default="0"/>

	<!-- Method # (A/B/C)) -->
	<arg name="method" default="A"/>

	<!-- velocity-based with trajopt planning PID -->
	<node name="pid_trajopt" pkg="iact_control" type="pid_trajopt.py" args="$(arg p_gain) $(arg i_gain) $(arg d_gain) $(arg ID) $(arg task) $(arg method)" output="screen"/>

</launch>
